contractor/edge_based_graph_factory.cpp

```diff
@@ -120,27 +120,27 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u,
 
         // reconstruct bidirectional edge with individual weights and put each into the NN index
 
-        std::vector<int> forward_dist_prefix_sum(forward_geometry.size(), 0);
-        std::vector<int> reverse_dist_prefix_sum(reverse_geometry.size(), 0);
+        std::vector<int> forward_weight_prefix_sum(forward_geometry.size(), 0);
+        std::vector<int> reverse_weight_prefix_sum(reverse_geometry.size(), 0);
 
         // quick'n'dirty prefix sum as std::partial_sum needs addtional casts
         // TODO: move to lambda function with C++11
-        int temp_sum = 0;
+        int temp_weight_sum = 0;
```

Comment:
The idea is that the weight is of the order of magnitude of the duration (so with an UB of ~100 days). However, the weight has now its own 32bits, so the actual UB is even higher.

If someone decides that the weight are 1% of the duration across the world, it might be a problem. I’m not sure where that should be checked.